src/api/notificationActions.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+
+import { NOTIFICATION_SUCCESS_STATUS, tBase } from '@/constants/constants';
+import axiosInstance from '@/utils/axiosInstance';
+import i18n from '@/utils/i18n';
+
+interface NotificationPayload {
+  type: string;
+  link_text: string;
+  link_href: string;
+  message: string | null;
+  userId: number | undefined;
+}
+
+export const createNotification =
+  (payload: NotificationPayload) => async (): Promise<void> => {
+    try {
+      const response = await axiosInstance.post('/notifications', payload);
+
+      if (response.status === NOTIFICATION_SUCCESS_STATUS) {
+        toast.success(i18n.t(`${tBase}.notifications.success`));
```

Comment:
why not just t()?

Reply:
I just did the same thing as in the code with requests to endpoints. It seems that both options have almost the same functionality. If I change to import { t } from 'i18next', then replace it in other files as well?

Reply:
You need to import only one method, not the entire module

Reply:
replaced